chirp/drivers/tdh8.py

```diff
@@ -1519,13 +1519,23 @@ def _get_settings(self):
         basic.append(rs)
 
         if self.MODEL != "RT-730":
-            rs = RadioSetting("rogerprompt", "Roger",
-                              RadioSettingValueBoolean(_settings.rogerprompt))
+            if self.MODEL in H3_LIST:
+                # H3 uses roger-beep list
+                rs = RadioSetting("rogerprompt", "Roger",
+                                  RadioSettingValueList(
+                                      self._roger_list,
+                                      current_index=_settings.rogerprompt))
+            else:
+                # H8 uses roger-beep bool
+                rs = RadioSetting("rogerprompt", "Roger",
+                                  RadioSettingValueBoolean(
+                                      _settings.rogerprompt))
+
             basic.append(rs)
 
         rs = RadioSetting("txled", "Disp Lcd(TX)",
-
                           RadioSettingValueBoolean(_settings.txled))
+
```

Comment:
Please try to avoid adding or removing random whitespace like this. It makes the lines look changed in the git history when they weren't, which makes it hard to figure out what/why something was changed years later.